itests/node_client/node_client.go

```diff
@@ -101,3 +101,21 @@ func (c *NodesClients) ClearBlackList(t *testing.T) {
 	c.GoClients.HttpClient.ClearBlackList(t)
 	c.ScalaClients.HttpClient.ClearBlackList(t)
 }
+
+func (c *NodesClients) WaitForConnectedPeers(t *testing.T, timeout time.Duration) (error, error) {
+	errGo := Retry(timeout, func() error {
+		cp, _, err := c.GoClients.HttpClient.ConnectedPeers(t)
+		if len(cp) == 0 && err == nil {
+			err = fmt.Errorf("no connected peers")
```

Comment:
Please use `errors.New()` here. Try not to use `fmt.Errorf`

Reply:
Done